std/src/main/scala/algebra/std/byte.scala

```diff
@@ -0,0 +1,58 @@
+package algebra
+package std
+
+import algebra.lattice._
+import algebra.number._
+import algebra.ring._
+import algebra.std.util.StaticMethods
+
+package object byte extends ByteInstances
+
+trait ByteInstances {
+  implicit val byteAlgebra = new ByteAlgebra
+
+  val ByteMinMaxLattice = new Lattice[Byte] {
```

Comment:
My only point is this, we seem to have a lot of the same code that is really doing this:

```scala
class OrderLattice[T](implicit ord: Order[T]) extends Lattice[T] {
  def join(x: T, y: T): T = ord.max(x, y)
  def meet(x: T, y: T): T = ord.min(x, y)
}
```
Why not write that once rather than write it for each primitive?